presto-main/src/main/java/com/facebook/presto/operator/index/IndexLoader.java

```diff
@@ -193,7 +192,7 @@ private IndexedData getIndexedDataForKeys(Page indexPage)
                     for (UpdateRequest request : requests) {
                         request.failed(t);
                     }
-                    Throwables.propagate(t);
+                    throw t;
```

Comment:
Change `Throwable t` to `RuntimeException e`?

Reply:
Same thing, it could be `Error`.